### ISSUE TITLE
[Meds / Improvements] Update medication names to a monospace font

### ISSUE THREAD

**Issue by michaelbrodsky-a6** [department-of-veterans-affairs/va.gov-team#122285]
## Blocker
Kay suggested that we pause this work, and mark it as blocked, until she gains additional clarity around **(a)** whether we in fact want to move forward with it; and **(b)** if so, what type of monospace font we'd want to use.

## Context
This work was requested by Kay via DSVA Slack on October 15, 2025

## Background
The [monospace font](https://design.va.gov/foundation/typography#monospace-fonts) was recently added to the VA design system and there is concurrence that this font should be used for Med names.

## Description
Update font for medication names to a "monospace" font.

## Resources
VA Design System Guidance: [linked](https://design.va.gov/foundation/typography#monospace-fonts)

## Acceptance Criteria
- [ ] Medication names font has been updated to a monospace font
- [ ] Peer review
